packages/good-design/src/core/buttons/LearnButton.tsx

```diff
@@ -1,61 +1,33 @@
 import React, { useCallback } from "react";
 import { Box, Text } from "native-base";
-import BasePressable from "./BasePressable";
 import { openLink } from "@gooddollar/web3sdk-v2";
+
+import BasePressable from "./BasePressable";
 import { Image } from "../images";
-import BillyWaiting from "../../assets/images/billy-waiting.png";
-import BillySign from "../../assets/images/billy-sign.png";
 
 export type learnSources = "send" | "bridging" | "network" | "sign" | "identity";
-type links = { link: string; label: string; icon: any };
 
-const linksNew: Record<learnSources, links> = {
-  send: {
-    link: "https://gooddollar.notion.site/How-do-transactions-work-in-web3-ccf11b4e16874a1682722a20c4e24742",
-    label: "How do transactions work?",
-    icon: BillyWaiting
-  },
-  bridging: {
-    link: "https://gooddollar.notion.site/What-is-bridging-f42ec5d3c388454bb2266411a67d93f5",
-    label: "How does bridging work?",
-    icon: BillyWaiting
-  },
-  network: {
-    link: "https://gooddollar.notion.site/What-is-a-web3-network-4bf6c8efecab4834b25c006f57687cc6",
-    label: "What is a web3 network?",
-    icon: BillyWaiting
-  },
-  sign: {
-    link: "https://gooddollar.notion.site/What-is-signing-b0019fe6c43241068050c9aa16e87ee1",
-    label: "What is signing?",
-    icon: BillySign
-  },
-  identity: {
-    link: "https://www.notion.so/gooddollar/Get-G-873391f31aee4a18ab5ad7fb7467acb3",
-    label: "Learn about the identification process",
-    icon: BillySign
-  }
+type LearnButtonType = {
+  icon?: any;
+  label?: string;
+  learnTitle?: string;
+  link?: string;
 };
 
-type sourceOrAlt = {
-  source?: learnSources;
-  altSource?: links;
-} & ({ source: learnSources } | { altSource: links });
-
-const LearnButton = ({ source, altSource }: sourceOrAlt) => {
-  const { link, label, icon } = source ? linksNew[source] : altSource;
-
+const LearnButton = ({ icon, label, learnTitle, link }: LearnButtonType) => {
   const openNotionTab = useCallback(async () => {
-    await openLink(link, "_blank");
+    if (link) {
+      await openLink(link, "_blank");
+    }
   }, [link]);
 
   return (
     <BasePressable variant="externalLink" onPress={openNotionTab} w="100%">
       <Box display="flex" w="60%" alignSelf="flex-start" p={2}>
         <Text color="lightBlue" fontSize="sm">
-          LEARN
+          {learnTitle}
```

Comment:
why are you passing this as variable?

Reply:
Because the learn button is exclusively used by/in modals, so you canot use the Trans object here. see other comment.

Reply:
you need to put the languageprovider above the nativebase provider and modals context should work fine.

Reply:
also can you simply use the `%il8%` comment?

Reply:
Right, yes that seems to do the trick (moving the languageprovider)